.github/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# Changes from release 2024/11 to 2024/12
+1. AIRAC (2412) - Correct Luton (EGGW) Tug Release Point V orientation - thanks to @AdriTheDev (Callum Hicks)
```

Comment:
```suggestion
1. AIRAC (2412) - Corrected Luton (EGGW) Tug Release Point V orientation - thanks to @AdriTheDev (Callum Hicks)
```
Per [Style Guide](https://github.com/VATSIM-UK/UK-Sector-File/blob/main/.github/Style%20Guide.md#entry-summary-), changelog entries should be past tense :)